docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
   - /topics/storage.md
 ---
 
-TODO: update per v2 (no longer any job cache)
+# Storage in Brigade
 
-# How Brigade uses Kubernetes Persistent Storage
+Brigade utilizes storage for its in the following ways:
 
-Brigade allows script authors to declare two kinds of storage:
+  * [Shared Worker storage](#shared-worker-storage) wherein a Brigade Worker's
+    workspace may be shared with its Job(s). Optional; not enabled by default.
+  * [Artemis storage](#artemis-storage) for Brigade's Messaging/Queue component
+  * [MongoDB storage](#mongodb-storage) for Brigade's backing data store
 
-- per-job caches, which persist across builds
-- per-build shared storage, which exists as long as the build is running
+## Shared Worker storage
 
-Usage of these is described within the [JavaScript docs](javascript.md) and the
-[scripting guide](scripting.md).
+The workspace for a Brigade Worker can be shared with all Worker Jobs. This is
+an opt-in feature and isn't enabled by default. When enabled, Brigade will
+create a [PersistentVolume] on the underlying Kubernetes substrate and
+automatically add the corresponding volume mount to each Worker Job created.
 
-This document describes the underlying Kubernetes architecture of these two
-storage types.
+> Note: As this volume may be accessed by more than one pod, and each pod may
+need both read and write access to the shared volume, its access mode is
+[ReadWriteMany][Access Modes], which may not be supported by the default
+[storage class] configured on your Kubernetes substrate. See the [Access Modes]
+matrix for compatibility. Brigade is well-tested using [NFS] and [Azure File]
+on [AKS]. ([Azure Disk] does *not* support this required access mode.)
 
-## Brigade and PersistentVolumeClaims
+To configure the storage class that should be used for shared Worker storage,
+set the `worker.workspaceStorageClass` field in Brigade's [Helm chart values]
+file to the name of the storage class. For example, if NFS is set up on the
+Kubernetes cluster and its storage class is named `nfs`, this would be the
+appropriate configuration:
```

Comment:
Worth mentioning that nfs is also the default in the chart?

Reply:
Oh, yes, I forgot it was the default.  I'll add a mention.